main.tf

```diff
@@ -42,7 +42,7 @@ locals {
  * Create a vault role that can be logged in to via an AWS IAM Role
  */
 resource "vault_aws_auth_backend_role" "role" {
-  for_each = var.login_role_arns ? toset(var.backend_paths) : []
+  for_each = length(var.login_role_arns) > 0 ? toset(var.backend_paths) : []
```

Comment:
I think [this](https://github.com/transcend-io/terraform-vault-role/pull/5/commits/b3f57576c4c320fa964d6d867c0f8373b29a91ff#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbL63) would also need to updated then?

Reply:
never mind, `enable_approle_login` is a boolean